service/elastigroup/providers/aws/aws.go

```diff
@@ -874,6 +874,7 @@ type Instance struct {
 	PrivateIP        *string    `json:"privateIp,omitempty"`
 	PublicIP         *string    `json:"publicIp,omitempty"`
 	CreatedAt        *time.Time `json:"createdAt,omitempty"`
+	Ipv6Address      *string    `json:"ipv6Address,omitempty"`
```

Comment:
please rename to:
IPv6Address      *string

Reply:
renamed